erigon-lib/txpool/fetch.go

```diff
@@ -187,18 +186,21 @@ func (f *Fetch) receiveMessage(ctx context.Context, sentryClient sentry.SentryCl
 			return err
 		}
 		if req == nil {
+			f.logger.Warn("[txpool.receiveMessage]", "req nil")
 			return nil
 		}
+		f.logger.Info("[txpool.receiveMessage]", "req", req)
 		if err := f.handleInboundMessage(streamCtx, req, sentryClient); err != nil {
 			if grpcutil.IsRetryLater(err) || grpcutil.IsEndOfStream(err) {
 				time.Sleep(3 * time.Second)
 				continue
 			}
-			f.logger.Debug("[txpool.fetch] Handling incoming message", "msg", req.Id.String(), "err", err)
+			f.logger.Debug("[txpool.fetch] Handling incoming message", "msg", string(req.Data), "reqID", req.Id.String(), "err", err)
 		}
 		if f.wg != nil {
 			f.wg.Done()
 		}
+		f.logger.Info("[txpool.fetch] Handling incoming message", "msg", string(req.Data), "reqID", req.Id.String(), "err", err)
```

Comment:
erigon handling tons of concurrent requests - from RPC and from P2P. no much reason to log them all.

Reply:
also this change is not related to current PR

Reply:
removed